### DISCUSSION TITLE
Wind generation model in ANDES

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#448] [Modeling] [answered]
The built-in case ieee14_wt3n.xlsx includes four wind-related models: WTDTA1 (wind turbine drive-train), WTARA1 (wind turbine aerodynamics), WTPTA1 (wind turbine pitch control), and WTTQA1 (wind turbine generator torque).
Are these models equivalent to the [WT3 - Generic Wind Model](https://www.esig.energy/wiki-main-page/wt3-generic-wind-model/#PSSE): WT3G (Generator/Converter), WT3P (Pitch control), WT3T (Mechanical control), WT3E (Electrical control)?

**Comment by cuihantao** [comment, marked as answer]
The second-generation models (WTDTA1, ATARA1, etc) are modified from the first-generation models (WT3G, WT3P, etc). Comparing the block diagrams will tell the differences, which I suppose are not major.
